Update _index.md (#72539)
grafana/grafana: docs/sources/setup-grafana/image-rendering/_index.md

To install the plugin, refer to the [Grafana Image Renderer Installation instructions](/grafana/plugins/grafana-image-renderer/?tab=installation#installation).

## Testing Image Rendering

> Once you have enabled image rendering, you can test if it is working by following these steps:

Reviewer: ```suggestion
Once you have enabled image rendering, complete the following steps to test it is working:
```

> * Go to a panel that you want to render as an image.

Reviewer: ```suggestion
1. Navigate to a panel that you want to render as an image.
```

> * Click the dropdown arrow next to the panel title.

Reviewer: ```suggestion
1. Click the dropdown arrow next to the panel title.
```

> * Click Share > Direct link rendered image.

Reviewer: ```suggestion
1. Click **Share > Direct link rendered image**.
```

> If image rendering is working correctly, you will see a link to the rendered image. You can click on the link to open the image in a new tab.

Reviewer: ```suggestion
If image rendering works correctly, you will see a link to the rendered image. You can click the link to open the image in a new tab.
```

  }
}
```

> ## Features Not Available in Grafana Cloud

Reviewer: ```suggestion
## Features not available in Grafana Cloud
```
